bakery/src/cli/execute.js

```diff
@@ -325,6 +325,47 @@ const tasks = {
       }
     }
   },
+  link: (parentCommand) => {
+    const commandUsage = 'link <collid> <canonicallist>'
+    const handler = async argv => {
+      const buildExec = path.resolve(BAKERY_PATH, 'build')
+
+      const imageDetails = imageDetailsFromArgs(argv)
+      const taskArgs = imageDetails == null
+        ? []
+        : [`--taskargs=${JSON.stringify(imageDetails)}`]
+      const taskContent = execFileSync(buildExec, ['task', 'link-book', ...taskArgs])
```

Comment:
The name `link-book` seems really vague to me. I know we brainstormed some potential options in Slack, and perhaps it's worth revisiting that conversation? (names are hard and subjective, I know, and perhaps `link-book` is the best option we can come up with)